doc/python/hover-text-and-formatting.md

```diff
@@ -312,21 +363,31 @@ continent_data = {continent:df_2007.query("continent == '%s'" %continent)
 
 fig = go.Figure()
 
-for continent_name, continent in continent_data.items():
-    fig.add_trace(go.Scatter(
-        x=continent['gdpPercap'],
-        y=continent['lifeExp'],
-        name=continent_name,
-        text=continent['continent'],
-        hovertemplate=
-        "<b>%{text}</b><br><br>" +
-        "GDP per Capita: %{x:$,.0f}<br>" +
-        "Life Expectation: %{y:.0%}<br>" +
-        "Population: %{marker.size:,}" +
-        "<extra></extra>",
-        marker_size=continent['size'],
+for continent_name, df in continent_data.items():
+    fig.add_trace(
+        go.Scatter(
+            x=df['gdpPercap'],
+            y=df['lifeExp'],
+            marker_size=df['size'],
+            name=continent_name,
+
+            # The next three parameters specify the hover text
+            # Text supports just one customized field per trace, but is simple to implement
+            text=df['continent'],  
+            # Custom data supports multiple fields through numeric indices in the hovertemplate 
+            # If I were not looking for an opportunity to demonstrate the text parameter, 
+            # I would likely just add continent as a third customdata field.
```

Comment:
```suggestion
            # Custom data supports multiple fields through numeric indices in the hovertemplate 
            # In we weren't using the text parameter in our example, 
            # we could instead add continent as a third customdata field.
```

@rl-utility-man just want to keep that language similar to the language earlier in the example. Does this still read as you intended?

Reply:
Looks great!